opencloudData/src/main/java/eu/opencloud/android/data/OpencloudDatabase.kt

```diff
@@ -50,6 +50,7 @@ import eu.opencloud.android.data.migrations.MIGRATION_35_36
 import eu.opencloud.android.data.migrations.MIGRATION_37_38
 import eu.opencloud.android.data.migrations.MIGRATION_41_42
 import eu.opencloud.android.data.migrations.MIGRATION_42_43
+import eu.opencloud.android.data.migrations.MIGRATION_48_49
```

Comment:
Why is there a 48.json and a 49.json?
For a single commit or PR, shouldn't it be one new version?